azerothcore/azerothcore-wotlk
# (apps/Db assembler) Re-make db_assembler script

## Changes Proposed:
-  Modify the current assembler system, since it is obsolete, and in reality, nobody uses it, because most of them enable the options in the emulator, so that it executes them for them. However, there is a small group of people, that sometimes, we prefer to correct the scripts manually, so it seemed to me a good idea, to generate a script, that allows only, to generate the files.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Windows 10

## How to Test the Changes:


1. Go to the folder `apps/db_assembler`
2. Execute the script.
3. Select menu options
4. The files should be generated within `env/dist/sql`

## Known Issues and TODO List:


## Screenshots

![menu](https://user-images.githubusercontent.com/2810187/213880446-c34f3683-e224-4473-9d0e-5b4f4a12093b.png)

![result](https://user-images.githubusercontent.com/2810187/213880449-b6864b29-0511-4e79-a71c-fc0081bff79f.png)


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
